packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -0,0 +1,41 @@
+import {Component, h, Host, Listen, State} from '@stencil/core';
+
+export interface UpdateLiveRegionEventArgs {
+  region: string;
+  message: string;
+}
+
+/**
+ * The `atomic-aria-live` component notifies screen readers of changes in the search interface.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-aria-live',
+  shadow: false,
+})
+export class AtomicAriaLive {
+  @State() public regions: Record<string, string> = {};
+
+  @Listen('atomic/accessibility/updateLiveRegion', {target: 'document'})
+  protected updateMessage({
+    detail: {message, region},
+  }: CustomEvent<UpdateLiveRegionEventArgs>) {
+    this.regions = {...this.regions, [region]: message};
+  }
+
+  public render() {
+    return (
+      <Host style={{position: 'absolute', right: '10000px'}}>
+        {Object.entries(this.regions).map(([region, message]) => (
```

Comment:
Why are there multiple regions vs. just one?

Reply:
Although screen readers reads both the old value of a region and its new one, I don't fully trust that every screen reader is going to do so correctly. This is mostly preventive. I don't want, for instance, an aria-live for a facet overwriting the query summary.

It would be fine to make all three of the current components use the same region though.

Another advantage with this approach, is that it's going to be a very easy change if we want one region to be `assertive`.